src/components/MultipieceModal/MultipieceModal.js

```diff
@@ -65,7 +65,7 @@ const MultipieceModal = (props) => {
         </Col>
         {
           (item.numberOfMissingPieces || item.missingPieces) &&
-          <Fragment>
+          <React.Fragment key="missingPieces">
```

Comment:
Could you please clarify the purpose of adding `key` to fragment? If the key is not needed then the syntax `<></>` of Fragment can be used instead.

Reply:
babel-loader can't resolve <> tag, so regarding existing documentation (https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-fragments.md) I did this.

Reply:
Fixed the issues by test my changes on new workspace